Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -205,6 +214,8 @@ def output_unbottled(formulae, deps_hash, noun, hash, any_named_args)
           when XcodeRequirement
             next true unless r.version
 
+            require "os/mac/xcode" if OS.linux?
```

Comment:
I didn't run `brew style` after https://github.com/Homebrew/brew/pull/14870/commits/7586c368cf989de2a4fbfef59dcc350cab960bc5 but it seems I have a style failure for exactly that change:
```
Library/Homebrew/dev-cmd/unbottled.rb:217:39: C: Move OS.linux? and OS.mac? calls to extend/os.
            require "os/mac/xcode" if OS.linux?
                                      ^^^^^^^^^
```
Would it be okay to unconditionally `require "os/mac/xcode"` (in a new `extend/os/linux/dev-cmd/unbottled.rb` file if required, which would then be `require`d in this file, similar to `update-test`)?